### DISCUSSION TITLE
How to upgrade to RC1

### DISCUSSION THREAD

**Discussion by chrpai** [wixtoolset/issues#7092] [Questions] [answered]
I see nuget getting updated packages and I'm eargly champing at the bit to try it.  How do we use the new version?  I don't see any project properties in HeatWave.  I tried updating the .wixproj manually but it just says:

```
<Project Sdk="WixToolset.Sdk/4.0.0-rc.1">`
  <ItemGroup>
    <PackageReference Include="WixToolset.Iis.wixext" Version="4.0.0-rc.1" />
    <PackageReference Include="WixToolset.UI.wixext" Version="4.0.0-rc.1" />
    <PackageReference Include="WixToolset.NetFx.wixext" Version="4.0.0-rc.1" />
  </ItemGroup>

```
C:\Github\Test\Installer\Test\Test.wixproj : warning MSB4240: Multiple versions of the same SDK "WixToolset.Sdk" cannot be specified. The previously resolved SDK version "4.0.0-preview.1" from location "C:\Github\Test\Installer\TestMM\TestMM.wixproj" will be used and the version "4.0.0-rc.1" will be ignored.

**Comment by robmen** [comment, marked as answer]
We're still in the process of updating everything. Wait until you see an announcement on [News](https://wixtoolset.org/news/) for WiX v4 or [FireGiant's blog](https://www.firegiant.com/blog/) for HeatWave to know that everything is in place.
